python/ribasim/pyproject.toml

```
@@ -30,10 +32,16 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 tests = ["pytest", "pytest-xdist", "pytest-cov", "ribasim_testmodels"]
+netcdf = ["xugrid"]
+delwaq = ["jinja2", "networkx", "netcdf"]
```

```suggestion
delwaq = ["jinja2", "networkx", "netCDF4"]
```

Should this be the PyPI name?
https://pypi.org/project/netCDF4/
https://pypi.org/project/netcdf/